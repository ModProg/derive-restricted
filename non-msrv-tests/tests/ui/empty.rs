#[derive(derive_where::DeriveWhere)]
#[derive_where()]
struct Test<T>(T);

fn main() {}
