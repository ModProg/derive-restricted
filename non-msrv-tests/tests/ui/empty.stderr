error: empty `derive_where` found
 --> tests/ui/empty.rs:2:3
  |
2 | #[derive_where()]
  |   ^^^^^^^^^^^^^^
